packages/terra-demographics-banner/src/DemographicsBannerDisplay.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ResponsiveElement from 'terra-responsive-element';
+import 'terra-base/lib/baseStyles';
+
+import './DemographicsBanner.scss';
+import SmallDemographicsBannerDisplay from './_SmallDemographicsBannerDisplay';
```

Comment:
I want to use this convention to show this is the private component. Private components are just used inside the module and never should be exported.